product-service/swagger.yaml

```diff
@@ -0,0 +1,40 @@
+openapi: 3.0.0
+info:
+  title: Online Fruit Stall API
+  description: API for Online Fruit Stall site.
+  version: 0.0.1
+servers:
+  - url: https://l6mg2h5d9g.execute-api.eu-west-1.amazonaws.com/dev/
+    description: Dev server for testing
+paths:
+  /products:
+    get:
+      summary: Returns a list of products.
+      description: Just a list of Products
+      responses:
+        '200':
+          description: A JSON array of products
+          content:
+            application/json:
+              schema: 
+                type: array
+                items: 
+                  type: string
+        '400':
+          description: "Bad request"
+  /products/{productId}:
+    get:
+      summary: Returns a product by Index.
+      description: Returns a product by Index.
+      responses:
+        '200':
+          description: A JSON object of a product
+          content:
+            application/json:
+              schema: 
```

Comment:
The descriptive schema for the Product should be provided.
```
{
  id: nmber,
  title: string;
  and ect.
}
```

Reply:
fixed it